storage/engine/mvcc.go

```diff
@@ -47,11 +47,7 @@ const (
 //  - Key count (count of all keys, including keys with deleted tombstones)
 //  - Value count (all versions, including deleted tombstones)
 //  - Intents (provisional values written during txns)
-type MVCCStats struct {
-	LiveBytes, KeyBytes, ValBytes, IntentBytes int64
-	LiveCount, KeyCount, ValCount, IntentCount int64
-	IntentAge, GCBytesAge, LastUpdateNanos     int64
-}
+type MVCCStats proto.MVCCStats
```

Comment:
Sadly, you can only use local types as a receiver.  So it will have to remain as an aliased type.  I might be able to pick and choose some of the instances where it appears, but I can't just do an outright replace.